borg/archiver.py

```diff
@@ -2033,24 +2032,25 @@ def prerun_checks(self, logger):
 
     def _setup_implied_logging(self, args):
         """ turn on INFO level logging for args that imply that they will produce output """
-        # map of option name to name of child logger (which matches the option name)
+        # map of option name to name of logger for that option
         option_logger = {
-                'output_list': 'list',
-                'show_version': 'show-version',
-                'show_rc': 'show-rc',
-                'stats': 'stats',
+                'output_list': 'borg.output.list',
+                'show_version': 'borg.output.show-version',
```

Comment:
hmm, there are no requirements on the logger name? like being a valid python name or so?

just asking because of that "-" in there and because usually module names are used for this.